app/views/users/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<%if @user.errors.any?%>
+  <p>The fields are mandotory</p>
+  <% @user.errors.full_messages.each do |e|%>
+    <ul style="color:red"><%= e %></ul>
+    <% end %>
+<% end%>
+
+
+<%= form_for @user do |f| %>
+  Username:</br><%= f.text_field :username, placeholder: "Your username" %></br>
+  Email:</br> <%= f.text_field :email, placeholder: "example@example.com" %></br>
+  Password:</br> <%= f.text_field :password, placeholder: "Your password" %></br>
+  </br><%= f.submit 'Submit'%></br>
+<% end %>
+
+
+
+<!--
+<form accept-charset="UTF-8" action="/users" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="user[username]"><br><br>
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="user[email]"><br><br>
+  <label for="password">Password:</label>
+  <input type="text" id="password" name="user[password]"><br><br>
+  <input type="submit" value="Submit">
+</form>
+-->
+
+<!-- 
+<%= form_tag("/users", method: "post") do %>
```

Comment:
It's better to have `users_path` instead of hardcoding the address `/users`. The first one will always direct the user to the right place even if the address changes.

There is no need to specify the method when posting since that's the default in this type of form.

Reply:
Thanks, Majo, for your review and backstage support! We really appreciate it.
We believe we have solved all the questions you raised.
Thanks.